docs/community/README.md

```diff
@@ -66,6 +66,17 @@ See [How to commit](HowToCommit.md) for helper doc for Submarine committers.
 
 Communication within the Submarine community abides by [Apache’s Code of Conduct](https://www.apache.org/foundation/policies/conduct.html).
 
+## Mailing lists
+
+Get help using Apache Submarine or contribute to the project on our mailing lists:
+
++ [Users](https://lists.apache.org/list.html?users@submarine.apache.org) : [subscribe](users-subscribe@submarine.apache.org), [unsubscribe](users-unsubscribe@submarine.apache.org), [archives](https://lists.apache.org/list.html?users@submarine.apache.org)
+for usage questions, help, and announcements.
++ [Dev](https://lists.apache.org/list.html?dev@submarine.apache.org) : [subscribe](dev-subscribe@submarine.apache.org), [unsubscribe](dev-unsubscribe@submarine.apache.org), [archives](https://lists.apache.org/list.html?dev@submarine.apache.org)
+for people wanting to contribute to the project.
++ [Commits](https://lists.apache.org/list.html?commits@submarine.apache.org) : [subscribe](commits-subscribe@submarine.apache.org), [unsubscribe](commits-unsubscribe@submarine.apache.org), [archives](https://lists.apache.org/list.html?commits@submarine.apache.org)
```

Comment:
The links for subscribing and unsubscribing should be "mailto:xxxxx@submarine.apache.org"